tests/core/pyspec/eth2spec/test/helpers/execution_payload.py

```diff
@@ -259,7 +259,7 @@ def build_randomized_execution_payload(spec, state, rng):
 
     num_transactions = rng.randint(0, 100)
     execution_payload.transactions = [
-        spec.Transaction(get_random_bytes_list(rng, rng.randint(0, 1000)))
+        get_random_tx(rng)
```

Comment:
This helper should probably condition on `is_post_bellatrix` (or whatever the appropriate helper is) to toss in some random blob TXs in Deneb and after

Reply:
I see